sceptre/cli/template.py

```diff
@@ -137,3 +138,32 @@ def fetch_remote_template_command(ctx, path):
             output.append(template)
 
     write(output, context.output_format)
+
+
+@click.command(name="detect-stack-drift", short_help="Detects stack drift on running stacks.")
+@click.argument("path")
+@click.pass_context
+@catch_exceptions
+def detect_stack_drift_command(ctx, path):
+    """
+    Detect stack drift on running stacks.
+    \f
+
+    :param path: The path to execute the command on.
+    :type path: str
+    """
+    context = SceptreContext(
+        command_path=path,
+        project_path=ctx.obj.get("project_path"),
+        user_variables=ctx.obj.get("user_variables"),
+        options=ctx.obj.get("options"),
+        ignore_dependencies=ctx.obj.get("ignore_dependencies")
+    )
+
+    plan = SceptrePlan(context)
+    responses = plan.detect_stack_drift()
+    output = "\n".join([
+        json.dumps({stack_name: response}, sort_keys=True, indent=2, default=str)
+        for stack_name, response in responses.values()
+    ])
```

Comment:
This should make a list of dicts and then output that using the output_format, rather than hard-coding json here.

Reply:
@jfalkenstein The issue here is that the JSON renderer is pretty simple, and in the case of this detect drifts feature, it produces a long JSON string that is unreadable without the sort of formatting I did here. Also, I just discovered that the yaml renderer is broken for this input. See the bug I raised [here](https://github.com/Sceptre/sceptre/issues/1173). What should we do?

Reply:
My recommendation would be to fix that loader, yeah. converting the string into a io.StringIO should be sufficient to use yaml.safe_load.

Reply:
Regardless of what you do, I think we should follow the dictates of the output_format. If the user has explicitly specified that, I don't see a reason why we should override that and always output json.